### PR TITLE
support k8s v1.19

### PR DESCRIPTION
**Pull request by Thor-wl**
Signed-off-by: Thor-wl <13164644535@163.com>

### CODE REVIEW

File: pkg/scheduler/actions/allocate/allocate.go [hunk 1]
```diff
@@ -272,3 +280,38 @@ func (alloc *Action) Execute(ssn *framework.Session) {
 }
 
 func (alloc *Action) UnInitialize() {}
+
+func setPodVolumesByNode(pod *v1.Pod, node *v1.Node) *volumescheduling.PodVolumes {
+	client := fake.NewSimpleClientset()
```
Comment (william-wang) [inline pkg/scheduler/actions/allocate/allocate.go:285]: why use fake ?

File: pkg/scheduler/actions/allocate/allocate.go [hunk 2]
```diff
@@ -272,3 +282,38 @@ func (alloc *Action) Execute(ssn *framework.Session) {
...
+		runtime.WithClientSet(client),
+		runtime.WithInformerFactory(informerFactory),
+	}
+	fh, err := runtime.NewFramework(nil, nil, nil, opts...)
+	if err != nil {
+		klog.V(3).Infof("new framework handle instance failed for %v", err.Error())
+		return nil, err
+	}
+	pl, err := volumebinding.New(&config.VolumeBindingArgs{
+		BindTimeoutSeconds: 300,
```
Comment (william-wang) [inline pkg/scheduler/actions/allocate/allocate.go:299]: if binding timeout happen, it will block other task for 300s. I think 60s is enough.

**Review by william-wang — APPROVED** [review]
/lgtm

**Comment by william-wang** [comment]
@Thor-wl The Job PVC e2e test case failed, please take a look.

**Comment by Thor-wl** [comment]
> @Thor-wl The Job PVC e2e test case failed, please take a look.

fixed

**Pull request closed without merging**